plugins/Course/bannerScraper.py

```diff
@@ -0,0 +1,116 @@
+from asyncio.windows_events import NULL
+import requests
+from bs4 import BeautifulSoup
+import datetime
+
+# The stuff between these lines was taken from https://github.com/jackharrhy/muntrunk/blob/master/muntrunk/scrape.py
+# Full credit to Jack for this stuff
+# --------------------------------------------------------------------------------------------------
+headers = {
+    "User-Agent": "github.com/cmoyates/MUN-Info-Discord-Bot",
```

Comment:
might wanna update this now that its being used in automata

Reply:
Updated